tox.ini

```diff
@@ -23,6 +23,13 @@ passenv =
 commands =
     py.test --cov=scrapy --cov-report= {posargs:--durations=10 docs scrapy tests}
 
+[testenv:typing]
+basepython = python3
+deps =
+    mypy==0.780
```

Comment:
Technically, last version is 0.781, but 0.780 is the last one to have an actual [release post](https://mypy-lang.blogspot.com/2020/06/mypy-0780-released.html) so I thought it might be safer.